kedro-datasets/kedro_datasets/spark/spark_dataset.py

```diff
@@ -26,6 +27,27 @@
 logger = logging.getLogger(__name__)
 
 
+def _get_spark() -> Any:
+    """
+    Returns the SparkSession, we need this wrapper because the SparkSession
+    is retrieved differently in databricks-connect vs a Notebook in web UI
```

Comment:
```suggestion
    Returns the SparkSession. In case databricks-connect is available we use it for
    extended configuration mechanisms and notebook compatibility,
    otherwise we use classic pyspark.
```